src/v2/Components/ArtworkFilter/ArtworkFilterArtworkGrid.tsx

```diff
@@ -127,3 +127,22 @@ export const ArtworkFilterArtworkGridRefetchContainer = createFragmentContainer(
     `,
   }
 )
+
+const LoadingArea: React.FC<{ isLoading?: boolean }> = ({
+  isLoading,
+  children,
+}) => {
+  return (
+    <Box position="relative">
+      {isLoading && (
+        <Sticky>
+          <Box height={300} width="100%" position="absolute">
+            <Spinner />
+          </Box>
+        </Sticky>
+      )}
+
+      <Box opacity={isLoading ? 0.1 : 1}>{children}</Box>
+    </Box>
+  )
```

Comment:
It's always bothered me that this wasn't a skeleton?

Reply:
Prob for a separate PR

Reply:
It bothers me that we have a component named skeleton. 👀